README.md

```diff
@@ -1 +1,129 @@
-# angular-forms-readme
\ No newline at end of file
+# Built-in form validation methods
+
+## Overview
+
+You might've been thinking that Angular didn't have much more to offer - you were wrong! Angular helps us out massively with a quite common issue a lot of developers have to deal with - forms!
+
+## Objectives
+
+- Describe Angular form validation and states
+- Write a submit function
+- Create multiple inputs with different validation types
+- Display state Objects in the View
+
+## Angular Forms
+
+We're already two-way binding our input values to our controller via `ng-model`, but Angular allows us to take our form integration much, much further.
+
+When we assign a `name` to our form and input elements, Angular begins its magic and gives us access to a lot of useful tools.
```

Comment:
Here, can you describe what the problem we're solving is: so far, we haven't been validating anything that the user has been inputting ie we could end up with lousy data in our controller.